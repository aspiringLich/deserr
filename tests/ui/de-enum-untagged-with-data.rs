use deserr::Deserr;

#[derive(Deserr)]
#[deserr(error = deserr::Error)]
enum Enum {
    EmptyVariant,
    VariantWithSomething { data: usize },
}

fn main() {}
