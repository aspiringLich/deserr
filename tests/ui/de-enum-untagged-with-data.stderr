error: Externally tagged enums are not supported yet by deserr. Add #[deserr(tag = "some_tag_key")]
 --> tests/ui/de-enum-untagged-with-data.rs:3:10
  |
3 | #[derive(Deserr)]
  |          ^^^^^^
  |
  = note: this error originates in the derive macro `Deserr` (in Nightly builds, run with -Z macro-backtrace for more info)
